index.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 const From = require('fastify-reply-from')
+const UrlData = require('fastify-url-data')
```

Comment:
Please don't use this module. It will slow things down considerably :(.

I think we should just identify if there is a querystring with a basic `indexOf('?')` and slice.